modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -361,6 +363,383 @@ Additionally, the user requires some `ReadOnly` permissions, and some of the rol
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+
+[discrete]
+[id="installation-vsphere-installer-infra--minimum-requirements_{context}"]
+== Minimum required vCenter account privileges
+
+After you create a custom role and assign privileges to it, you can create permissions by selecting specific vSphere objects and then assigning the custom role to a user or group for each object. 
+
+Before you create permissions or request for the creation of permissions for a vSphere object, determine what minimum permissions apply to the vSphere object. By doing this task, you can ensure a basic interaction exists between a vSphere object and {product-title} architecture. 
+
+[IMPORTANT]
+====
+If you create a custom role and you do not assign privileges to it, the vSphere Server by default assigns a `Read Only` role to the custom role. Note that for the cloud provider API, the custom role only needs to inherit the privileges of the `Read Only` role.
+====
+
+Consider creating a custom role when an account with global administrative privileges does not meet your needs.
+
+[IMPORTANT]
+====
+Red Hat supports only an account that is configured with the required privileges. Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected behaviors that might occur when creating a custom role with a restricted set of privileges. 
```

Comment:
This phrasing is sitting oddly with me. I think changing "supports only" to "only supports" would help, but here's another suggestion to take or leave:
```suggestion
Accounts that are not configured with the required privileges are unsupported. Installing an {product-title} cluster in a vCenter is tested against a full list of privileges as described in the "Required vCenter account privileges" section. By adhering to the full list of privileges, you can reduce the possibility of unexpected behaviors that might occur when creating a custom role with a restricted set of privileges. 
```

Reply:
Good suggestion. I was trying to remove much passive voice as possible but I was wrestling with the word "only"!

Reply:
I think this (<thing> is unsupported) is one of the times when passive voice makes more sense, somehow. But I appreciate your effort to avoid it :)